tools/compress_annovar_output.cwl

```diff
@@ -0,0 +1,28 @@
+cwlVersion: v1.2
+class: CommandLineTool
+id: sort_gzip_index_vcf
+doc: "Quick tool to sort, compress and index a vcf file"
+requirements:
+  - class: ShellCommandRequirement
+  - class: InlineJavascriptRequirement
+  - class: ResourceRequirement
+    ramMin: 16000
+    coresMin: 8
+  - class: DockerRequirement
+    dockerPull: 'pgc-images.sbgenomics.com/d3b-bixu/autopvs1:v0.2.0a'
```

Comment:
should this be 2.0a? https://github.com/JiguangPeng/autopvs1/tree/v2.0

Reply:
Created a PR here: https://github.com/d3b-center/bixtools/pull/192 - although lots of mods to the original which begs the question:
1. Do I treat it as revision to the original?
2. Give it a whole new version since it's a port?
3. Actually call it v1 since we are happy in prod and it's a modified port of the original?
Thoughts @dmiller15 ?